editor/connections_dialog.cpp

```diff
@@ -1227,7 +1227,23 @@ void ConnectionsDock::_slot_menu_about_to_popup() {
 	slot_menu->set_item_disabled(slot_menu->get_item_index(SLOT_MENU_DISCONNECT), connection_is_inherited);
 }
 
-void ConnectionsDock::_rmb_pressed(const Ref<InputEvent> &p_event) {
+void ConnectionsDock::_gui_input(const Ref<InputEvent> &p_event) {
+	// Handle Delete
+	if (ED_GET_SHORTCUT("connections_editor/disconnect")->matches_event(p_event)) {
```

Comment:
```suggestion
	if (ED_IS_SHORTCUT("connections_editor/disconnect", p_event)) {
```

Reply:
Can I just press "commit suggestion" or should I try and keep it in one commit?

Reply:
Eh I'll just amend it :)